plugin/adsk/scripts/AETemplateHelpers.py

```diff
@@ -22,3 +23,16 @@ def GetRootLayerName(proxyShape):
     except:
         pass
     return ''
+
+def ProxyShapePayloads(proxyShape, loadAll):
+    try:
+        proxyPath = ufe.PathString.path(proxyShape)
+        ufeItem = ufe.Hierarchy.createItem(proxyPath) if proxyPath else None
+        if ufeItem:
+            contextOps = ufe.ContextOps.contextOps(ufeItem)
+            if contextOps:
+                cmdStr = 'Load with Descendants' if loadAll else 'Unload'
+                cmd = contextOps.doOpCmd([cmdStr])
+                ufeCmd.execute(cmd)
```

Comment:
Use the context ops to either load or unload all. This makes the action undoable.